setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "k3s"
+description: Set up a Coder deployment with a Ubuntu + k3s
+---
+
+k3s is a lightweight Kubernetes distribution that works well for single-node or
+multi-node clusters alike.
+
+This guide covers installation on a new Ubuntu 20.04 LTS machine. If you are
+looking to install Coder on a local machine or an existing host,
+[Kind](./kind.md) or k3d may be a better choice.
+
+> This install method is not officially supported or tested by Coder. If you
+> have questions or run into issues, feel free to reach out on our
+> [community Slack channel](https://cdr.co/join-community).
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the following:
+
+1. **Ubuntu 20.04 machine**: This can be a bare metal or virtual machine. For
+   cloud compute, AWS EC2, GCP Compute Engine, Azure VMs, Vultr, DigitalOcean
+   work well. Make sure the machine specs satisfies Coder's
+   [resource requirements](../requirements.md)
+1. Network policy or firewall accepting incoming traffic on ports 22 (SSH), 80
+   (HTTP), 443 (HTTPS), 5349 (Networking v2 / TURN), and 8443 (Kubernetes API).
+1. The following software installed on the machine:
+   - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+   - [helm](https://helm.sh/docs/intro/install/)
+
+### Resource allocation and performance
+
+Your experience with Coder is dependent on your system specs. See our
+[resource requirements](../requirements.md) for more details.
+
+## Change the default SSH port
+
+If you want to allow
+[SSH into workspaces](https://coder.com/docs/coder/v1.20/workspaces/ssh), you
+will need to change the host's default SSH port to free up port 22. You may also
+need to modify your firewall to accept incoming traffic from the alternative
+port (e.g 5522)
+
+[Follow this guide](https://linuxize.com/post/how-to-change-ssh-port-in-linux/)
+by Linuxize if you do not know how to do this.
+
+## Install k3s with Calico
+
+The following uses
+[Calico's quickstart guide](https://docs.projectcalico.org/getting-started/kubernetes/k3s/quickstart)
+to set up K3s. We will be disabling the default network policies and Traefik in
+favor of Calico and nginx-ingress.
```

Comment:
I'm not clear on which default network policies -- K3s network policies?

Reply:
yes I think describing them as "k3s network policies" is most accurate.